recipes/ms_dotnet3.rb

```diff
@@ -20,10 +20,10 @@
 
 if platform?('windows')
   include_recipe 'ms_dotnet'
-
-  if win_version.windows_server_2008? || win_version.windows_server_2008_r2? || win_version.windows_7? || win_version.windows_vista?
+  if win_version.windows_server_2012? || win_version.windows_server_2012_r2? || win_version.windows_server_2008? || win_version.windows_server_2008_r2? || win_version.windows_7? || win_version.windows_vista?
     windows_feature 'NetFx3' do
       action :install
+      all true
```

Comment:
Thank you Annih for the technical note. I'll update the code to do a condition on the "all" based on the Windows version.